pycqed/measurement/calibration/two_qubit_gates.py

```diff
@@ -1418,6 +1450,6 @@ def run_analysis(self, analysis_kwargs=None, **kw):
         if 'TwoD' not in analysis_kwargs['options_dict']:
             analysis_kwargs['options_dict']['TwoD'] = True
         self.analysis = tda.MultiQubit_TimeDomain_Analysis(
-            qb_names=[task['qbr'] for task in self.task_list],
+            qb_names=list(self.exp_metadata['meas_obj_sweep_points_map'].keys()),
```

Comment:
This feels less transparent. Is this the standard way of accessing this information?

Reply:
Part of the PR was to read out both qubits instead of reading out only one (formerly called qbr). I have now found an easier way of implementing the new version. Moreover, I found some remnants of qbr in other functions, which I now cleaned up.